Makefile

```diff
@@ -11,23 +8,37 @@ setup_helm_repos:
 	helm repo add codacy-external https://charts.codacy.com/external
 	helm repo update
 
-.PHONY: update_versions
-update_versions:
+.PHONY: create_version_file
+create_version_file:
+	@if [ ! -e .version ]; \
+	then echo "Creating .version file based on the most recent tag..."; \
+	git tag --sort creatordate | grep -E "^([0-9]+\.[0-9]+\.[0-9]+)-RC.*" | tail -n -1 > .version; \
+	cat .version; \
+	else echo ".version file already exists..."; \
+	fi
```

Comment:
New makefile target to create the version file.
This is mostly to be used when doing the initial manual steps of commencing a release from a local machine.